kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/converter/RecordConverter.kt

```diff
@@ -82,22 +82,23 @@ abstract class RecordConverter(override val sourceType: String, val avroData: Av
 
         val sourceRecords = contents.asSequence().map contentMap@{
```

Comment:
`asSequence` is not needed when there is just one operation and the operations are finalised with `toList`. It is useful if the following conditions are met:
1. There are a lot of objects.
2. There are multiple subsequent `map`, `filter`, etc. operations or the end result is not a list.